core/src/main/java/org/bitcoinj/core/KeyId.java

```diff
@@ -57,19 +82,16 @@ protected void parse() throws ProtocolException {
 
     @Override
     protected void bitcoinSerializeToStream(OutputStream stream) throws IOException {
-
         stream.write(bytes);
     }
 
-    public String toString()
-    {
-        return "KeyId(" + Utils.HEX.encode(bytes) +")";
+    public String toString() {
+        return "KeyId(" + Utils.HEX.encode(Utils.reverseBytes(bytes)) +")";
     }
```

Comment:
As with `Sha256Hash` (a 32 byte hash value), `KeyId` (a 20 byte hash) is also displayed in big endian by Dash Core RPC commands, so we will do the same.